backend/tests/src/lib/helpers/random/choose/choose.helper.ts

```diff
@@ -0,0 +1,8 @@
+const choose = <T>(array: T[]): T => {
```

Comment:
Let's provide more clear naming for methods, eg (or something like that)
```suggestion
const getRandomElementFromArray = <T>(array: T[]): T => {
```